website/content/en/docs/deploy-guide.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Development Guide"
+linkTitle: "Development Guide"
+weight: 50
+---
+
+# Resources Karpenter Expects/Creates
+
+- Cluster on AWS
+    - min version 1.16? 1.18?
+    - must be EKS?
+- IAM Roles
+    - `KarpenterNode` -- run containers, configure networking
+    - `Karpenter` -- provision new instances
+    - IRSA vs kube2iam?
+- Service Accounts
+    - `Karpenter` -- associated with `Karpenter` IAM Role
+- Tagged Subnets? 
```

Comment:
We discover subnets in an eks agnostic way. Same for security groups. This is actually standard for upstream k8s w/ the aws cloud provider.

`tagkey: kubernetes.io/cluster/$name`